### ISSUE TITLE
Error starting the manager

### ISSUE THREAD

**Issue by lfqpi** [optimato/lab-control-lib#11]
Hi,

I am testing lab-control-lib on a computer running Windows 11 with python 3.12.0.
I have a problem when I try to start the manager from ipython. 
After importing dummylab, I give the following command, based on the tutorial.

```
 manager = dummylab.lclib.manager.ManagerBase.Server()
```
The difference, with respect to the tutorial, is that I had to replace Manager with ManagerBase.

The output is the following:

```
TypeError                                 Traceback (most recent call last)
Cell In[10], line 1
----> 1 manager = dummylab.lclib.manager.ManagerBase.Server()

File ~\Software\LabControlSystem\lab-control-lib\lclib\proxydevice.py:749, in ProxyServerBase.__init__(self, instantiate, instance_args, instance_kwargs)
    746 self.service = self._create_service()
    748 # Start serving
--> 749 self.activate()

File ~\Software\LabControlSystem\lab-control-lib\lclib\proxydevice.py:758, in ProxyServerBase.activate(self)
    752 """
    753 Start serving
    754 """
    755 # Create rpyc threaded server
    756 self.rpyc_server = ThreadedServer(
    757     service=self.service,
--> 758     port=self.ADDRESS[1],
    759     protocol_config={
    760         "allow_all_attrs": True,
    761         "allow_setattr": True,
    762         "allow_delattr": True,
    763     },
    764     disconnect_callback=self.del_client,
    765 )
    767 # Replace print and input
    768 self.logger.info("Rerouting 'print' and 'input'")

TypeError: 'NoneType' object is not subscriptable
```

Is it the correct way of starting the manager?

Thank you

**Comment by pierrethibault** [comment]
I'm sorry, I didn't update the documentation. It should just be 

`dummylab.Manager.Server()`

(or from the command line: `python -m lclib dummylab start manager`)

**Comment by lfqpi** [comment]
Thank you Pierre.

This starts the manager without errors, but creates a new problem.
Even if the manager is running in a different terminal, the user interface doesn't find it.

From ipython:

```
dummylab.ui.init()
```
gives the following error:

```shell_session
In [2]: dummylab.ui.init()
[24/10/2024 16:52:52] [ERROR] [lclib.MonitorProxyClient] Connection to ('localhost', 5001) refused. Is the server running?
[24/10/2024 16:52:52] [INFO] [lclib] Connection failed
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
Cell In[2], line 1
----> 1 dummylab.ui.init()

File ~\Software\LabControlSystem\lab-control-lib\lclib\ui\ui.py:47, in init(yes, manager_name)
     45 monitor_client = client_or_None('monitor', client_name=client_name)
     46 if monitor_client is None:
---> 47     raise RuntimeError('Could not connect to monitor! Is it running?')
     48 drivers['monitor'] = monitor_client
     50 # Experiment management

RuntimeError: Could not connect to monitor! Is it running?
```
When I list the ports (netstat under Windows), I don't see any process listening on port 5001.
I will investigate further during the week and I will let you know if I find the source of the problem.

**Comment by pierrethibault** [comment]
TLDR: You need both `Manager` and `Monitor` to be running.

Explanation: That's something relatively new I implemented: What used to be called just `Manager` is now split into two: `Monitor` is the unique main hub that needs to be launched first: it monitors all other drivers and grabs metadata when asked to. `Manager` is actually the "Experiment manager", which takes care of scan numbering and file naming. There is only one `Manager` currently in the dummylab example, but I split it in such a way that many managers can run simultaneously. This is needed for setups (like mine) where two experiments can run at the same time: different detectors will get their file naming and scan numbering from different managers to avoid clashes. This is not completely integrated in the ui, but in the future it will be possible to choose which Manager to talk to in an interactive session.

**Comment by lfqpi** [comment]
Thank you very much for your help.
I started the Manager and Monitor and now it works: I can move the dummy motor.
